demo/index.tsx

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable no-console */
 import React, { useState, useEffect, useCallback } from "react";
 import { createRoot } from "react-dom/client";
-import { RLN, genExternalNullifier } from "rlnjs";
+import { RLN } from "rlnjs";
 import { bigintToHex } from "bigint-conversion";
 import { Identity } from "@semaphore-protocol/identity";
+import { formatBytes32String } from "@ethersproject/strings";
```

Comment:
You don't need this dependency and you can use `encodeBytes32String` from `ethers`.